package.json

```diff
@@ -60,6 +60,7 @@
     "@types/node": "^10.12.2",
     "@types/semver": "^5.5.0",
     "all-contributors-cli": "^6.0.0",
+    "babel-code-frame": "^6.26.0",
```

Comment:
what did you add this dependency for?

Reply:
It was development dependency for tslint and is not anymore. In "packages\typescript-estree\tests\ast-alignment\parse.ts" it is referred without having it in dependencies.